students/16th/team2/jmyoon/westagram/user/views.py

```diff
@@ -1,5 +1,74 @@
 import json
+import re
+import bcrypt
+import jwt
 
-from django.http import JsonResponse
-from django.views import View
+from django.http   import JsonResponse
+from django.views  import View
 
+from user.models   import User
+from my_settings   import SECRET_KEY
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        # email 정규표현식
+        vali_email = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        validate_email=re.compile(vali_email)
+
+        try:
+            if not data['email']:
+                return JsonResponse(
+                    {'MESSAGE' : 'KeyERROR'}, status = 400)
+            
+            if not validate_email.match(data['email']):
+                return JsonResponse(
+                    {'MESSAGE' : '잘못된 email형식'}, status = 400)
+
+            if len(data['password']) < 8 :
+                return JsonResponse(
+                    {'MESSAGE' : 'password는 8자리 이상'}, status = 400)
+
+            if (User.objects.filter(email = data['email']).exists()):
```

Comment:
- exists() 활용 너무 좋습니다!
- 소괄호는 불필요하니 없애주세요~
```suggestion
            if User.objects.filter(email = data['email']).exists():
```